src/modules/dashboard/components/Members/MembersTitle.css

```diff
@@ -49,10 +49,10 @@
 }
 
 .icon {
-  height: 30px;
-  width: 30px;
+  height: 13.33px;
```

Comment:
The icon should actually be 20 px as per figma file. The 13.3 px is only if you select the circle only.
Generally, we try to never use decimals for pixels as well.
<img width="1097" alt="Screenshot 2022-06-10 at 18 27 06" src="https://user-images.githubusercontent.com/34057551/173110273-b7aa3e2b-8d84-4ef6-8238-aea6df7356a2.png">